skunkworks_crow/src/main/java/org/odk/share/views/ui/settings/SettingsActivity.java

```diff
@@ -162,4 +169,20 @@ private void showPasswordDialog() {
         alertDialog.setCancelable(true);
         alertDialog.getWindow().setSoftInputMode(WindowManager.LayoutParams.SOFT_INPUT_STATE_ALWAYS_VISIBLE);
     }
+
+    private void resetSettings() {
+        AlertDialog.Builder builder = new AlertDialog.Builder(this);
+        builder.setTitle(getString(R.string.title_reset_settings))
+                .setMessage(getString(R.string.message_reset_settings))
+                .setPositiveButton(getString(R.string.ok), (DialogInterface dialog, int which) -> {
+                    SharedPreferences.Editor editor = prefs.edit();
+                    editor.clear();
```

Comment:
This reset option is supposed to reset application, now we've to clear db's and tables as well.